src/LiquidityPool.sol

```diff
@@ -462,10 +464,27 @@ contract LiquidityPool is Initializable, OwnableUpgradeable, UUPSUpgradeable, IL
         emit Rebase(getTotalPooledEther(), eETH.totalShares());
     }
 
+    function payProtocolFees(int128 _protocolFees) public {
```

Comment:
Unless we consider particular cases of penalizing the protocol (we dont have it as of now at least)
we don't need to consider the negative protocol fee which leads to `burnShares` in the below flow.

For now, let's make it simple and consider only positive protocol fee flows.

If any slashing happens and we have to slash eETH, `_protocolFees` will be `0`.
Slashing on eETH will penalize the protocol's eETH holdings equally, anyway